lib/rdoc/ri/driver.rb

```diff
@@ -744,7 +744,7 @@ def complete name
     complete_klass  name, klass, selector, method, completions
     complete_method name, klass, selector,         completions
 
-    completions.sort.uniq
+    completions.sort.uniq.select {|s| s.start_with? name }
```

Comment:
Just a suggestion: maybe call `.sort` last would be slightly more efficient given the list size being smaller?